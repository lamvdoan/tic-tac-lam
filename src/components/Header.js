import React from 'react';
import { Link } from 'react-router-dom';

const Header = () => (
    <header className="header">
        <div className="content-container">
            <Link className="header__title" to="/">
                <h1>Tic Tac Lam</h1>
            </Link>
        </div>
    </header>
);
export default Header;
