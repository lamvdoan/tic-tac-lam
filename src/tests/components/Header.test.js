import React from 'react';
import Header from '../../components/Header';
import { shallow } from 'enzyme';

test('should render header properly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();

    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
});