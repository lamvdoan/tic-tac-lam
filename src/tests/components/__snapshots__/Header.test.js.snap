// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header properly 1`] = `
<header
  className="header"
>
  <div
    className="content-container"
  >
    <Link
      className="header__title"
      replace={false}
      to="/"
    >
      <h1>
        Boilerplate
      </h1>
    </Link>
  </div>
</header>
`;
